src/Build/Microsoft.Build.csproj

```diff
@@ -30,6 +30,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Framework\Microsoft.Build.Framework.csproj" />
+    <ProjectReference Include="..\Utilities\Microsoft.Build.Utilities.csproj" />
```

Comment:
This is the only part of the PR that I'm worried about. It's a change in our assembly dependency graph and thus our package dependency graph, and it might result in unexpected loading of the Utilities assembly on new paths.

Can you split this PR into the things that don't need this and the things that do? I think we should take the former and wait on the latter--probably pulling it in for 16.2 preview 1.